### PR TITLE
fix: Set message sender to macros excution

### LINKED ISSUE #5731: Messages sent by the macro are shown as sent by the bot.

### PR DESCRIPTION
**Pull request by tejaswinichile**
## Description

- For event send_message and send_attachment set message sender to the macro executer

Fixes #5731 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Added test cases for the same

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by tejaswinichile** [comment]
https://www.loom.com/share/8988b6e6ec9c483fae5f746affb77a30

File: spec/controllers/api/v1/accounts/macros_controller_spec.rb [hunk 1]
```diff
@@ -249,7 +249,6 @@
     context 'when it is an authenticated user' do
       it 'execute the macro' do
         expect(conversation.messages).to be_empty
-        expect(conversation.assignee).to be_nil
```
Comment (pranavrajs) [inline spec/controllers/api/v1/accounts/macros_controller_spec.rb:252]: @tejaswinichile Why was this line removed?

Comment (tejaswinichile) [reply spec/controllers/api/v1/accounts/macros_controller_spec.rb:252]: Oh because it was failing in rubocop :D Too many lines in one spec.

**Review by pranavrajs — APPROVED** [review]

**Comment by pranavrajs** [comment]
@muhsin-k Can you please this before merging the changes?

File: spec/controllers/api/v1/accounts/macros_controller_spec.rb (hunk 1 above)
Comment (tejaswinichile) [reply spec/controllers/api/v1/accounts/macros_controller_spec.rb:252]: Added the specs here for sender check https://github.com/chatwoot/chatwoot/pull/5769/files#diff-05bb0b4908bb3017214fab118b732e88f575364d2ed97930bbec973d4c70a49eR261

**Comment by muhsin-k** [comment]
Tested in local environment and working as expected.

https://www.loom.com/share/2a96ae8b244347cbb4ee5dd60d2e3707

**Review by muhsin-k — APPROVED** [review]

**Pull request merged**